site: update FAQ (#11119)
kubernetes/minikube: site/content/en/docs/faq/_index.md

  Questions that come up regularly
---

> ## How to run an older Kubernetes version with minikube ?

Reviewer: ```suggestion
## How to run an older Kubernetes version with minikube?
```

You do not need to download an older minikube to run an older kubernetes version.

> You can create a Kubenretes cluster with any version you desire using `--kubernetes-version` flag.

Reviewer: ```suggestion
You can create a Kubernetes cluster with any version you desire using `--kubernetes-version` flag.
```

Example:

```bash
minikube start --kubernetes-version=v1.15.0
```

> ## Docker Driver: How to run minikube change cgroup manager used by minikube?

Reviewer: ```suggestion
## Docker Driver: How to set the cgroup manager minikube uses?
```

By default minikube uses the `cgroupfs` cgroup manager for the Kubernetes clusters, if you are on a system with a systemd cgroup manager, this could cause conflicts.
To use `systemd` cgroup manager, run:

```bash
minikube start --force-systemd=true
```

> ## How to run minikube with Docker driver if existing cluster is VM?

Reviewer: ```suggestion
## How to run minikube with Docker driver if existing cluster is using a VM driver?
```

If you have an existing cluster with a VM driver (virtualbox, hyperkit, KVM,...).

> First please ensure your Docker service is running and then you need to either delete the existing cluster and create one

Reviewer: ```suggestion
First, ensure your Docker service is running, then chose one of the following options:

Option 1: Delete the existing cluster and create a new one.
```

```bash
minikube delete
minikube start --driver=docker
```

> Alternatively, if you want to keep your existing cluster you can create a second cluster with a different profile name. (example p1)

Reviewer: ```suggestion
Option 2: Keep your existing cluster by creating a second cluster with a different profile name (example p1).
```
